### DISCUSSION TITLE
What should I do to make the badge wrap all the text?

### DISCUSSION THREAD

**Discussion by jvsdurso-pixaflow** [saadeghi/daisyui#3006] [Q&A] [answered]
![Captura de tela de 2024-04-29 10-54-49](https://github.com/saadeghi/daisyui/assets/167534901/dcf5b77f-a686-46dc-862f-511894fcf873)

I've tried adding `text-wrap` and changed the `overflow` parameter but nothing seems to work. Any ideias on this?

```ts
<Header className="text-wrap overflow-hidden overflow-ellipsis whitespace-nowrap mb-0"
                title={course?.titulo ?? "Aguarde..."}
                subtitle={modulo?.titulo ?? ""}
 />
 
```
 
```ts
<div>
          <h1 className={`text-wrap font-bold text-2xl ${className}`}>{title}</h1>
          {subtitle && (
            <div className="badge badge-ghost mt-3">
              <h2 className={`font-normal text-sm ${className}`}>{subtitle}</h2>
            </div>
          )}
</div>
```

**Comment by saadeghi** [comment, marked as answer]
Well, that text doesn't fit in that space, whether if it's badge or not.  
You can use Tailwind CSS utility class [`truncate`](https://tailwindcss.com/docs/text-overflow#truncate) to cut the string like this:  
https://play.tailwindcss.com/cWsFinffS5


Let me know if you have a question.
